stories/SlowPageAlert.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { Location } from 'history';
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import SlowPageAlert, { PublicProps } from '../src/components/SlowPageAlert';
+import { createFakeLocation } from '../src/test-helpers';
+
+const render = (otherProps: Partial<PublicProps> = {}) => {
+  const props = {
+    getMessage: (allowSlowPages: boolean) => 'example message',
+    getLinkText: (allowSlowPages: boolean) => 'example link text',
+    location: createFakeLocation(),
+    ...otherProps,
+  };
+  return <SlowPageAlert {...props} />;
+};
+
+storiesOf('SlowPageAlert', module).add('default', () => {
+  return render({
+    getMessage: (allowSlowPages: boolean) =>
+      'This file has been shortened for performance.',
+    getLinkText: (allowSlowPages: boolean) => 'Show the original file.',
```

Comment:
I think these two messages could be the default messages in `render()` so that you could simply `return render()`.